pmaweb/static/images/brand/phpMyAdmin-logo-preview.svg

```diff
@@ -0,0 +1,6 @@
+<svg fill="none" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 1020 792">
```

Comment:
This beautiful icon would be accepted into our data repository. You can open a pull-request to add new assets anytime :)

Reply:
I have a friend that is doing a fine tune on that logo, as soon she finishes, I'll do the PR with the brand guide.